src/components/SpeciesTable.js

```diff
@@ -384,14 +391,19 @@ const EditModal = ({
   };
 
   const handleSave = async () => {
-    setIsEdit(false);
-    await editSpecies({
-      id: speciesEdit.id,
-      name: speciesEdit.name,
-      desc: speciesEdit.desc,
-    });
-    loadSpeciesList();
-    setSpeciesEdit(undefined);
+    if (nameSpecies.includes(speciesEdit.name.trim())) {
```

Comment:
Can we make this a case-insensitive comparison?

Reply:
Yes. I have fixed in my latest commit.